keyboards/plexus75/keymaps/via/keymap.c

```diff
@@ -0,0 +1,94 @@
+        _______, _______, KC_APP,  _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_PSCR, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, RESET,   _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_MUTE, KC_VOLD, KC_VOLU, KC_BSLS, _______, _______,
+        RESET, RESET, _______, _______, _______, _______, _______, _______, _______, _______, _______
+    ),
+
+    /* 2nd Function Layer
+     * .-------------------------------------------------------------------------------------------------------------------------------------'
+     * |        |        |RGB ON OFF|HUE INC|HUE DEC|SAT INC |SAT DEC |        |        |        |        |        |        |       |PRNSCRN |
+     * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-------+--------|
+     * | `      | 1      | 2      | 3      | 4      | 5      |        |        |        | 6      | 7      | 8      | 9      |  0    |        |
+     * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+----------------|
+     * |  ~     | !      |  @     |  #     |  $     |  %     |        |        |        |  ^     |  &     |  *     |  (     |  )    | BCKSPC |
+     * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+----------------|
+     * |        |        |        |        |        |        |        |        |        | MUTE   | VOL DN | VOL UP | \      |       |        |
+     * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+-----------------+--------+----------------|
+     * |             | RESET  | RESET  |        |        |        |        |        |        |        |        |        |                    |
+     * '-------------------------------------------------------------------------------------------------------------------------------------'
+     */
+    [_FN1] = LAYOUT_2x2u(
+        _______, _______, RGB_TOG, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, _______, _______, _______, _______, _______, _______, _______, KC_PSCR,
+        KC_GRV,     KC_1,   KC_2,     KC_3,    KC_4,    KC_5, _______, _______, _______,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0, _______,
+        KC_TILD, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, _______, _______, _______, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_BSPC,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_MUTE, KC_VOLD, KC_VOLU, KC_BSLS, _______, _______,
+        RESET, RESET, _______, _______, _______, _______, _______, _______, _______, _______, _______
+    ),
+
+};
```

Comment:
Unless you've set the number of layers to be less than 4, you need to have 4 layers defined in the keymap, or the layer will be filled with gibberish. 

```suggestion
    [3] = LAYOUT_2x2u(
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
    ),
};
```

Reply:
will do

Reply:
This is still outstanding. Otherwise LGTM.

Reply:
Sorry. Github noob. Think I've fixed it now.